app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split(" ")
+    if "mv" not in command or len(command) != 3:
+        return
+
+    if "/" not in command[2]:
+        os.rename(command[1], command[2])
```

Comment:
Not fixed
Use unpacking (for example `a, b = 3, 4`) instead of indexes to access command parts